### PR TITLE
Add feature flaws to DG and add use case to DG

### PR DESCRIPTION
**Pull request by chuakid**
1. Added our two planned feature flaw fixes to dg
2. Added use case 17: View Pending Meetings

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -923,3 +933,9 @@ testers are expected to do more *exploratory* testing.
 | **Import a meeting**         | `importm VALID_JSON`                                                                                                                                                                                                 |
 | **Sort meetings**            | `sortm SORT_FIELD [r]` <br> e.g., `sortm dt/`                                                                                                                                                                        |
 
+
+## Appendix: Planned Enhancements
+1. Currently, the UI shows a blank area when an attribute is undefined for contacts and meetings.
+    2. Proposed Fix: Have a placeholder `No ATTRIBUTE defined` for each attribute.
+3. Currently, once we define an attribute for a contact, we are unable to remove it.
+    4. Proposed Fix: Add a new command "deleteattribute" for this purpose.
```
Comment (KS2499) [inline docs/DeveloperGuide.md:941]:
```suggestion
1. Currently, the UI shows a blank area when an attribute is undefined for contacts and meetings.
    1.1. Proposed Fix: Have a placeholder `No ATTRIBUTE defined` for each attribute.
2. Currently, once we define an attribute for a contact, we are unable to remove it.
    2.1. Proposed Fix: Add a new command "deleteattribute" for this purpose.
```
would this numbering would make more sense?

**Review by KS2499 — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -923,3 +933,9 @@ testers are expected to do more *exploratory* testing.
 | **Import a meeting**         | `importm VALID_JSON`                                                                                                                                                                                                 |
 | **Sort meetings**            | `sortm SORT_FIELD [r]` <br> e.g., `sortm dt/`                                                                                                                                                                        |
 
+
+## Appendix: Planned Enhancements
+1. Currently, the UI shows a blank area when an attribute is undefined for contacts and meetings.
+    1.1. Proposed Fix: Have a placeholder `No ATTRIBUTE defined` for each attribute.
+2. Currently, once we define an attribute for a contact, we are unable to remove it.
+    2.1. Proposed Fix: Add a new command "deleteattribute" for this purpose.
```
Comment (Beebeeoii) [inline docs/DeveloperGuide.md:941]: can we give a concrete example for this? for eg `deleteattribute p/1 e/` to remove email for a contact or `deleteattribute m/1 des/` to remove description for a meeting.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -923,3 +933,9 @@ testers are expected to do more *exploratory* testing.
 | **Import a meeting**         | `importm VALID_JSON`                                                                                                                                                                                                 |
 | **Sort meetings**            | `sortm SORT_FIELD [r]` <br> e.g., `sortm dt/`                                                                                                                                                                        |
 
+
+## Appendix: Planned Enhancements
+1. Currently, the UI shows a blank area when an attribute is undefined for contacts and meetings.
+    1.1. Proposed Fix: Have a placeholder `No ATTRIBUTE defined` for each attribute.
```
Comment (Beebeeoii) [inline docs/DeveloperGuide.md:939]:
```suggestion
    1.1. Proposed Fix: Have a placeholder `No ATTRIBUTE defined` for each undefined attribute.
```

**Review by Beebeeoii — CHANGES_REQUESTED** [review]
some minor edits to enhance clarity

**Review by Beebeeoii — APPROVED** [review]
lgtm thanks !

**Pull request merged**